Dockerfile.aarch64

```diff
@@ -20,7 +20,7 @@ ENV HOME="/config" \
   DUPLICATI__WEBSERVICE_ALLOWED_HOSTNAMES=*
 
 RUN \
-  echo "**** install packages ****" && \
+  echo "**** install pockages ****" && \
```

Comment:
Typo

Reply:
I'm not even sure how that typo is possible, but fixed